python/lsst/obs/decam/__init__.py

```diff
@@ -1,6 +1,5 @@
 from .version import *
-from .decamMapper import *
+
```

Comment:
Does this blank line need to be here?

Reply:
Maybe not, but I don't want to mess with the working ticket...